lib/ostiary/controller_helper.rb

```diff
@@ -28,15 +28,19 @@ def required_application_role(role, actions = {})
       # Exclude action(s) from requiring a role
       #  except: [*actions]
       # By default a given role will be required for every action
-      #
+      # Override role checking by passing a proc as method;
+      #   ostiary_policy method: :master?.to_proc, only: :show
       # One line creates one policy, which are immediately created with the proper class
-      def ostiary_policy(role, actions = {})
+      def ostiary_policy(role = nil, only: nil, except: nil, method: nil)
```

Comment:
We could break backwards compatibility with:
`def ostiary_policy(role: nil, only: nil, except: nil, method: nil)`

and then create a v1.x to show that

Reply:
nah